### ISSUE TITLE
Navigation with arrow keys in DropdownMenu

### ISSUE THREAD

**Issue by hansenalexander** [digdir/designsystemet#1688]
### Description of the bug

Currently testing an implementation of DropdownMenu in our application. JAWS (2023) suggests that I can use arrow up/down to navigate the menu, which does not work. Experiencing the same behavior in the [documentation](https://storybook.designsystemet.no/?path=/docs/felles-dropdownmenu--docs).

### Steps To Reproduce

1. Try navigating any of the [dropdown menus](https://storybook.designsystemet.no/?path=/docs/felles-dropdownmenu--docs) using JAWS.

**Comment by mimarz** [comment]
Thanks for reporting this! 

We'll have a look into this to make sure JAWS interprets the component correctly. 

We intentionally did not include up/down arrow navigation for now as its quite debatable as to wether a menu should support arrowkeys or not, based on usage. If its an overflow menu or not etc. i don't remember all the specific but I remember we did a lot of research at the time and were back and forth on this before ending up with not supporting up/down arrow keys.

**Comment by Barsnes** [comment]
Hi, as @mimarz mentioned above, we decided to not have arrow key navigation.
This was mostly because the buttons and links are added next in the taborder, and we felt that was more natural, combined with the point made about the debate about dropdownmenus and navigation.

We will have a look into making JAWS interpet this correctly, but if you have any good points to why arrow keys should be used, feel free to start a discussion!

**Comment by eirikbacker** [comment]
Will document this decision in https://github.com/digdir/designsystemet/issues/1482
